perf/istio/values-istio-sds-auth.yaml

```diff
@@ -0,0 +1,165 @@
+# This is used to generate istio.yaml for a stress/load test cluster.
+# It is generated based on values-istio-test.yaml, with SDS related setting.
```

Comment:
Don’t copy the file into a new file.
Just add the nodeagent section to the original file

Reply:
Removed the content copied from perf/istio/values-istio-test.yaml. Only the SDS related config is in the perf/istio/values-istio-sds-auth.yaml. values-istio-test.yaml and values-istio-sds-auth.yaml are in separate files because the former is for testing with SDS disabled and the latter is for testing with SDS enabled.